### PR TITLE
Content and user badging

### PR DESCRIPTION
**Pull request by anujtech46**
1- Content badge search
2- Assing badge to content
3- Profile badge

### CODE REVIEW

File: src/app/helpers/environmentVariablesHelper.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ let envVariables = {
   PORTAL_PRIMARY_BUNDLE_LANGUAGE: env.sunbird_portal_primary_bundle_language || 'en',
   CONTENT_SERVICE_UPSTREAM_URL: env.sunbird_content_service_upstream_url || 'http://localhost:5000/',
   LEARNER_SERVICE_UPSTREAM_URL: env.sunbird_learner_service_upstream_url || 'http://localhost:9000/',
-  DATASERVICE_URL: env.sunbird_dataservice_url || 'https://staging.open-sunbird.org/api/',
+  DATASERVICE_URL: env.sunbird_dataservice_url || 'https://dev.open-sunbird.org/api/',
```
Comment (harishgilimi) [inline src/app/helpers/environmentVariablesHelper.js]: Don't change this env variable

File: src/app/private/scripts/controllers/admin/adminController.js [hunk 2]
```diff
@@ -71,6 +70,8 @@ angular.module('playerApp')
                     .includes('SYSTEM_ADMINISTRATION')
                 }
               }
+              // Add batches mock response
```
Comment (harishgilimi) [inline src/app/private/scripts/controllers/admin/adminController.js]: badges not batches

File: src/app/private/views/badge/profileBadge.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<div class="ui raised segment margin-bottom-0" ng-if="isprofileinfo">
+    <!-- <span class="header cardsHeading">Certifactions & Awards</span>
```
Comment (harishgilimi) [inline src/app/private/views/badge/profileBadge.html]: Remove commented html

File: src/app/helpers/environmentVariablesHelper.js (hunk 1 above)
Comment (anujtech46) [reply src/app/helpers/environmentVariablesHelper.js]: Done.

File: src/app/private/scripts/controllers/admin/adminController.js (hunk 2 above)
Comment (anujtech46) [reply src/app/private/scripts/controllers/admin/adminController.js]: Done

File: src/app/private/views/badge/profileBadge.html (hunk 3 above)
Comment (anujtech46) [reply src/app/private/views/badge/profileBadge.html]: Done

**Pull request merged**